operator/cmd/mesh/operator-init.go

```diff
@@ -46,6 +46,7 @@ func addOperatorInitFlags(cmd *cobra.Command, args *operatorInitArgs) {
 	cmd.PersistentFlags().StringVar(&args.context, "context", "", ContextFlagHelpStr)
 	cmd.PersistentFlags().StringVar(&args.common.hub, "hub", hub, HubFlagHelpStr)
 	cmd.PersistentFlags().StringVar(&args.common.tag, "tag", tag, TagFlagHelpStr)
+	cmd.PersistentFlags().StringVar(&args.common.imagePullSecrets, "imagePullSecrets", "", ImagePullSecretsHelpStr)
```

Comment:
isn't this a list?

Reply:
Ah I see you split it. I think cobra allows StringSlice natively

Reply:
thanks for reminder, updated it to use `StringSlice`.